### PR TITLE
Added Badge Assign, Profile and Levels - Refs #10651

### PR DESCRIPTION
**Pull request by jloguercio**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -39,4 +39,4 @@ RewriteRule ^courses/([^/]+)/course-pic.png$ app/courses/$1/course-pic.png [QSA,
 RewriteRule ^session/(\d{1,})/about/?$ main/session/about.php?session_id=$1 [L]
 
 # Issued badges friendly URL
-RewriteRule ^badge/(\d{1,})/user/(\d{1,}) main/badge/issued.php?skill=$1&user=$2 [L]
+RewriteRule ^badge/issue/(\d{1,})/user/(\d{1,})/? main/badge/issued.php?issue=$1&user=$2 [L]
```
Comment (ywarnier) [inline .htaccess]: You cannot change previous rules like that. You have to maintain previous redirects just in case someone is still pointing at the old URL, and mention that this is the "old" version.

File: main/admin/skill.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<?php
+
```
Comment (ywarnier) [inline main/admin/skill.php]: Missing license terms and PHPDoc file block

File: main/admin/skill_level.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * For license terms, see /license.txt
```
Comment (ywarnier) [inline main/admin/skill_level.php]: The license terms need to be on a first, separate line from the PHPDoc block

File: main/admin/skill_profile.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * For license terms, see /license.txt
```
Comment (ywarnier) [inline main/admin/skill_profile.php]: The license terms need to be on a first, separate line from the PHPDoc block

File: main/badge/assign.php [hunk 5]
```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * For license terms, see /license.txt
```
Comment (ywarnier) [inline main/badge/assign.php]: The license terms need to be on a first, separate line from the PHPDoc block

File: src/Chamilo/CoreBundle/Entity/Skill.php [hunk 6]
```diff
@@ -2,6 +2,7 @@
```
Comment (ywarnier) [inline src/Chamilo/CoreBundle/Entity/Skill.php]: Missing license and PHPDoc block

File: src/Chamilo/CoreBundle/Entity/SkillRelUser.php [hunk 7]
```diff
@@ -7,7 +7,14 @@
 /**
```
Comment (ywarnier) [inline src/Chamilo/CoreBundle/Entity/SkillRelUser.php]: Missing license and PHPDoc block

File: src/Chamilo/CoreBundle/Entity/SkillRelUserComment.php [hunk 8]
```diff
@@ -0,0 +1,175 @@
+<?php
+/* For licensing terms, see /license.txt */
```
Comment (ywarnier) [inline src/Chamilo/CoreBundle/Entity/SkillRelUserComment.php]: Missing PHPDoc block

File: src/Chamilo/SkillBundle/Entity/Level.php [hunk 9]
```diff
@@ -0,0 +1,160 @@
+<?php
+/* For licensing terms, see /license.txt */
+
```
Comment (ywarnier) [inline src/Chamilo/SkillBundle/Entity/Level.php]: Missing PHPDoc block

File: src/Chamilo/SkillBundle/Entity/Profile.php [hunk 10]
```diff
@@ -0,0 +1,130 @@
+<?php
+/* For licensing terms, see /license.txt */
+
```
Comment (ywarnier) [inline src/Chamilo/SkillBundle/Entity/Profile.php]: Missing PHPDoc block

File: .htaccess (hunk 1 above)
Comment (ywarnier) [inline .htaccess]: I need an answer here... why this change?

**Comment by jloguercio** [comment]
Cierro este PR: he creado una nueva rama que contiene todas las correcciones y mejoras en aquí : https://github.com/chamilo/chamilo-lms/pull/1163
